index.html

```diff
@@ -0,0 +1,147 @@
+<!doctype html>
+<html>
+<head>
+	<meta charset="utf-8">
+
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Angular Application</title>
+
+	<style>
```

Comment:
Can you move these into a separate stylesheet? I think it will be easier to read